clearpath_sensors/config/ouster_os1.yaml

```diff
@@ -0,0 +1,117 @@
+    # - RNG19_RFL8_SIG16_NIR16
+    # - RNG15_RFL8_NIR8
+    # - RNG19_RFL8_SIG16_NIR16_DUAL
+    # - FUSA_RNG15_RFL8_NIR8_DUAL
+    udp_profile_lidar: ''
+    # metadata[optional]: path to save metadata file to, if left empty a file
+    # with the sensor hostname or ip will be crearted in ~/.ros folder.
+    metadata: ''
+    # lidar_port[optional]: port value should be in the range [0, 65535]. If you
+    # use 0 as the port value then the first avaliable port number will be
+    # assigned.
+    lidar_port: 0
+    # imu_port[optional]: port value should be in the range [0, 65535]. If you
+    # use 0 as the port value then the first avaliable port number will be
+    # assigned.
+    imu_port: 0
+    # sensor_frame[optional]: name to use when referring to the sensor frame.
+    sensor_frame: os_sensor
+    # lidar_frame[optional]: name to use when referring to the lidar frame.
+    lidar_frame: os_lidar
+    # imu_frame[optional]: name to use when referring to the imu frame.
+    imu_frame: os_imu
+    # point_cloud_frame[optional]: which frame of reference to use when
+    # generating PointCloud2 or LaserScan messages, select between the values of
+    # lidar_frame and sensor_frame.
+    point_cloud_frame: os_lidar
+    # pub_static_tf[optional]: when this flag is set to True, the driver will
+    # broadcast the TF transforms for the imu/sensor/lidar frames. Prevent the
+    # driver from broadcasting TF transforms by setting this parameter to False.
+    pub_static_tf: false
+    # proc_mask[optional]: use any combination of the 6 flags IMG, PCL, IMU, SCAN
+    # RAW and TLM to enable or disable their respective messages.
+    proc_mask: IMU|PCL|SCAN|IMG|RAW|TLM
+    # scan_ring[optional]: use this parameter in conjunction with the SCAN flag
+    # to select which beam of the LidarScan to use when producing the LaserScan
+    # message. Choose a value the range [0, sensor_beams_count).
+    scan_ring: 0
+    # use_system_default_qos[optional]: if false, data are published with sensor
+    # data QoS. This is preferrable for production but default QoS is needed for
+    # rosbag. See: https://github.com/ros2/rosbag2/issues/125
+    use_system_default_qos: false
+    # point_type[optional]: choose from: {original, native, xyz, xyzi, xyzir}
+    # Here is a breif description of each option:
+    #  - original: This uses the original point representation ouster_ros::Point
+    #          of the ouster-ros driver.
+    #  - native: directly maps all fields as published by the sensor to an
+    #          equivalent point cloud representation with the additon of ring
+    #          and timestamp fields.
+    #  - xyz: the simplest point type, only has {x, y, z}
+    #  - xyzi: same as xyz point type but adds intensity (signal) field. this
+    #          type is not compatible with the low data profile.
+    #  - xyzir: same as xyzi type but adds ring (channel) field.
+    #          this type is same as Velodyne point cloud type
```

Comment:
Is this reference to Velodyne intentional, or just a copy & paste error?

Reply:
This is coming directly from the Ouster driver itself.